java/src/main/java/com/konfigthis/splitit/client/ApiClient.java

```diff
@@ -240,6 +240,30 @@ public ApiClient setBasePath(String basePath) {
         return this;
     }
 
+    /**
+     * Get token path
+     *
+     * @return Token path
+     */
+    public String getTokenPath() {
+        return tokenPath;
+    }
+
+    /**
+     * Set token path
+     *
+     * @param tokenPath Token URL for oauth (e.g https://id.production.splitit.com/connect/token)
```

Comment:
this comment might be hard to add to template.

Reply:
this is already generated from the template 😎

Reply:
oh i see, yeah its fine then.